.github/workflows/ci-dev-contracts.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           version: nightly
 
       - name: Run tests
-        run: forge test -vvv
+        run: forge clean && forge test -vvv
```

Comment:
Given that this will be on a freshly mounted drive on a VM spun up to do all the tasks, why is a `clean` necessary? If a build took place earlier in the GitHub Action, then it is still a clean build. Why clean and force the `test` to build again?